monitoring/stats_history_test.cc

```diff
@@ -561,14 +561,90 @@ TEST_F(StatsHistoryTest, PersistentStatsReadOnly) {
   Close();
 
   // Reopen and flush memtable.
-  Reopen(options);
+  ASSERT_OK(TryReopen(options));
   Flush();
   Close();
   // Now check keys in read only mode.
   ASSERT_OK(ReadOnlyReopen(options));
 }
 #endif  // !ROCKSDB_LITE
 
+TEST_F(StatsHistoryTest, ForceManualFlushStatsCF) {
+  Options options;
+  options.create_if_missing = true;
+  options.write_buffer_size = 1024 * 1024 * 10;  // 10 Mb
+  options.max_background_jobs = 0;
```

Comment:
Setting `max_background_jobs` to 0 does not disable background flush, thus I think this line can be removed.